forslund/mycroft
# Include Precise and wake word model in Snap

This adds Precise as it's own part consisting of the latest Precise executable and wake word model. 

During mycroft-launch we check if they exist in the currently used directory `~/.mycroft/precise`. If they don't exist, they are unpacked from the Snap.

#### To Test
Build Snap, install and run - Precise should be available immediately and no download should be initiated.
